Client/src/Pages/Details/index.jsx

```diff
@@ -0,0 +1,156 @@
+    createData(<StatusLabel status="Cannot resolve" customStyles={{ backgroundColor: '#f2f4f7', borderColor: '#d2d6de', color: '#344054' }} />, '2024-03-14 21:41:09', 'timeout of 48000ms exceeded'),
+    createData(<StatusLabel status="Down" customStyles={{ backgroundColor: '#fff9f9', borderColor: '#ffcac6', color: '#344054' }} />, '2024-03-14 21:41:09', 'HTTP 350 - NOK'),
+  ];
+
+  return (
+    <Box
+      sx={{
+        marginTop: theme.spacing(4),
+        marginX: 'auto',
+        width: '80%',
+        paddingX: theme.spacing(6),
+        [theme.breakpoints.down('md')]: {
+          width: '100%',
+        },
+      }}
+    >
+      <Box sx={{ display: 'flex', justifyContent: 'space-around', marginBottom: theme.spacing(4) }}>
+        {[...Array(4)].map((_, index) => (
+          <Box
+            key={index}
+            sx={{
+              width: '223.35px',
+              height: '87px',
+              borderRadius: '4px',
+              border: '1px solid',
+              borderColor: '#EAECF0',
+              padding: theme.spacing(1),
+              textAlign: 'left',
+              boxShadow: '0px 1px 2px rgba(16, 24, 40, 0.05)',
+              backdropFilter: 'blur(2px)',
+            }}
+          >
+            <Typography
+              variant="body2"
+              sx={{
+                fontFamily: 'Roboto',
+                fontWeight: 400,
+                fontSize: '16px',
+                lineHeight: '24px',
+                color: '#1570EF',
+                marginBottom: theme.spacing(1),
+              }}
+            >
+              Currently up for
+            </Typography>
+            <Typography
+              variant="body1"
+              sx={{
+                fontFamily: 'Roboto',
+                fontWeight: 600,
+                fontSize: '13px',
+                lineHeight: '20px',
+                color: '#344054',
+              }}
+            >
+              4h 30m 2s
+            </Typography>
+          </Box>
+        ))}
+      </Box>
+      <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center', marginBottom: theme.spacing(2) }}>
+        <Typography variant="h6" component="div" sx={{ fontWeight: 600, fontSize: 16 }}>
+          History
+        </Typography>
+      </Box>
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 700 }} aria-label="customized table">
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>Status</StyledTableCell>
+              <StyledTableCell align="right">Date & Time</StyledTableCell>
+              <StyledTableCell align="right">Message</StyledTableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {rows.map((row, index) => (
+              <StyledTableRow key={index}>
```

Comment:
Index for keys [makes me sad ](https://react.dev/learn/rendering-lists#rules-of-keys) :cry:

Reply:
You told me about this one before. My bad. I will pay more attention next time. Sometimes I just take MUI's components and put them in here as is.

Reply:
No problem, it's a pet peeve of mine :joy:  Most of these will be removed as we update the components, but I'm always paranoid one will slip by us into production somewhere and leave us with a hard to track down bug

Reply:
Ya that's totally valid. We should definitely pay attention to it now before things get out of hand.